extension/sql/timescale-prometheus.sql

```diff
@@ -159,4 +159,78 @@ AS $func$
     VALUES ('timescale_prometheus_' || meta_key,meta_value, send_telemetry)
     ON CONFLICT (key) DO UPDATE SET value = EXCLUDED
 $func$
-LANGUAGE SQL VOLATILE SECURITY DEFINER;
\ No newline at end of file
+LANGUAGE SQL VOLATILE SECURITY DEFINER;
+
+----------------------------  aggregation functions ----------------------------
+
+CREATE FUNCTION @extschema@.prom_delta_transition(state internal, lowest_time timestamptz,
+    greatest_time timestamptz, step bigint, range bigint,
+    timev timestamptz, val double precision
+    )
```

Comment:
nit: no newline before `)`. Sorry my OCD was kicking in.

Reply:
I philosophically disagree 😉 
I think with multi-line arguments `( )` should be formatted as you would `{ }`